cmd/kubeadm/app/cmd/alpha/certs.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2018 The Kubernetes Authors.
+Copyright 2019 The Kubernetes Authors.
```

Comment:
apparently, the project does not change dates of old source files.
2018 should be kept here.

Reply:
til

Reply:
updated!